library/keystone_user.py

```diff
@@ -50,7 +50,7 @@
      description:
         - The keystone url for authentication
      required: false
-     default: 'http://127.0.0.1:35358/v2.0/'
+     default: 'http://127.0.0.1:35357/v2.0/'
```

Comment:
Oh, you already added it there. Then I'm even more confused as to why this module is being changed.